charts/xp-management/templates/_helpers.tpl

```diff
@@ -144,7 +144,7 @@ SegmenterConfig:
     MinS2CellLevel: 10
     MaxS2CellLevel: 14
 MLPConfig:
-  URL: {{ .Values.deployment.apiConfig.MlpConfig.URL | default (include "management-svc.mlp.server.url" .) | quote }}
+  URL: {{ include "management-svc.mlp.server.url" . | quote }}
```

Comment:
In Merlin and Turing, the MLP URL gets set like this:
* https://github.com/caraml-dev/helm-charts/blob/main/charts/merlin/templates/merlin-deployment.yaml#L281
* https://github.com/caraml-dev/helm-charts/blob/main/charts/turing/templates/_helpers.tpl#L203

On the other hand, the one in `xp-management` is a bit different: https://github.com/caraml-dev/helm-charts/blob/main/charts/xp-management/templates/_helpers.tpl#L115

Shall we adopt the same method as merlin / turing here too?

Reply:
Updated it. Thanks for pointing out 🙏